### ISSUE TITLE
0.9.1 doesn't work?

### ISSUE THREAD

**Issue by UpCoder** [TheWidlarzGroup/react-native-video#425]
if I execute npm install, then will happen a error as follow:
```
No compatible version found: react-native-video@^0.9.1
npm ERR! Valid install targets:
npm ERR! 0.9.0, 0.8.0, 0.8.0-rc, 0.8.0-beta, 0.7.1, 0.7.0, 0.6.1, 0.6.0, 0.5.1, 0.5.0, 0.4.12, 0.4.11, 0.4.10, 0.4.9, 0.4.8, 0.4.7, 0.4.5, 0.4.4, 0.4.3, 0.4.2, 0.4.1, 0.4.0, 0.3.6, 0.3.5, 0.3.3, 0.3.2, 0.3.1, 0.2.9, 0.2.8, 0.2.7, 0.2.6, 0.2.5, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.2.0, 0.1.9, 0.1.8, 0.1.6, 0.1.5, 0.1.4, 0.1.3, 0.1.2, 0.1.0
```
my packages.json file as follow:
```
{
  "name": "sampleProject",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest",
    "bundle-android": "react-native bundle --platform android --dev false --entry-file index.android.js --bundle-output android/app/src/main/assets/index.android.bundle --sourcemap-output android/app/src/main/assets/index.android.map --assets-dest android/app/src/main/res/"
  },
  "dependencies": {
    "co": "^4.6.0",
    "react": "15.4.1",
    "react-native": "0.39.0",
    "react-native-file-download": "0.0.10",
    "react-native-fs": "^2.0.1-rc.2",
    "react-native-navbar": "^1.5.4",
    "react-native-qiniu": "^0.2.2",
    "react-native-vector-icons": "^3.0.0",
    "react-native-video": "^0.9.1"
  },
  "devDependencies": {
    "babel-jest": "17.0.2",
    "babel-preset-react-native": "1.9.0",
    "jest": "17.0.3",
    "react-test-renderer": "15.4.1"
  },
  "jest": {
    "preset": "react-native"
  }
}
```
I want to update to 0.9.1 because the callback function(such as onEnd onLoad) can't be triggered with IOS10, "react-native-video": "^0.9.1" is wrong? how can i fix it? thank you very much😄

**Comment by mwiley** [comment]
The fix hasn't been released as a new version. You can change your package.json like this:

     "react-native-video": "git+https://github.com/react-native-community/react-native-video.git#master"

**Comment by UpCoder** [comment]
thanks for replay, it has fixed. but the callback functions still can't be called when I update the version(react native video) up 0.9.1. So sad. Finally,  I adjust the version of react and react native, then it's ok, the callback function can be called. package version as follow:
```
{
  "name": "sampleProject",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest",
    "bundle-android": "react-native bundle --platform android --dev false --entry-file index.android.js --bundle-output android/app/src/main/assets/index.android.bundle --sourcemap-output android/app/src/main/assets/index.android.map --assets-dest android/app/src/main/res/"
  },
  "dependencies": {
    "co": "^4.6.0",
    "react": "15.3.1",
    "react-native": "^0.33.0",
    "react-native-file-download": "0.0.10",
    "react-native-fs": "^2.0.1-rc.2",
    "react-native-navbar": "^1.5.4",
    "react-native-qiniu": "^0.2.2",
    "react-native-vector-icons": "^3.0.0",
    "react-native-video": "git+https://github.com/react-native-community/react-native-video.git#master"
  },
  "devDependencies": {
    "babel-jest": "17.0.2",
    "babel-preset-react-native": "1.9.0",
    "jest": "17.0.3",
    "react-test-renderer": "15.4.1"
  },
  "jest": {
    "preset": "react-native"
  }
}

```

**Comment by cq0702** [comment]
@mwiley  The version of my react native is 0.39.1 and I change my package.json like this:
 "react-native-video": "git+https://github.com/react-native-community/react-native-video.git#master". But it doen't work.I try npm install and it stucked.

**Comment by UpCoder** [comment]
maybe you need wait some minutes.  or use some vpn tools?
and you can try degrade the version of react

**Comment by larryranches** [comment]
@cq920702 Check issue #401 for a solution.

We have a PR out to bump the current version to have the new changes #419

**Comment by ijsnow** [comment]
Does this fix `onProgress` and other event handlers not being fired?

**Comment by larryranches** [comment]
@ijsnow Commit #381 fixes this which is in Master. I have a PR #419 to bump this project's version to have those commits. For a temp solution, you can update your package.json to pull from master.

**Comment by ghost** [comment]
Fixed with RNV 1.0
